vulnerabilities/hash-collision.md

```diff
@@ -0,0 +1,110 @@
+In Solidity, the `abi.encodePacked()` function is used to create tightly packed byte arrays which can then be hashed using `keccak256()`
```

Comment:
Should have a title at the top here

Reply:
How about
"Hash Collision when using `abi.encodePacked()` with Multiple Variable-Length Arguments"